### PR TITLE
lesson1 SQL file has been implemented

### PR DESCRIPTION
**Pull request by Hani-far**

### CODE REVIEW

File: databases/week1/Home work lesson 1.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+SELECT count(id) AS TotalTasks
+FROM task;
+
+-- 2.Find out how many tasks in the task table do not have a valid due date
+SELECT count(id) AS TotalTasks
+FROM task
+WHERE task.due_date is NULL;
+
+-- 3.Find all the tasks that are marked as done
+SELECT task.id, task.title, status.name As status
```
Comment (sboutzen) [inline databases/week1/Home work lesson 1.sql]: Avoid shadowing the `status` name. Instead you could call it statusName, which would avoid confusion with the table name status.

File: databases/week1/Home work lesson 1.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+WHERE status.name = "Done";
+
+-- 4.Find all the tasks that are not marked as done
+SELECT task.id, task.title, status.name As status
+FROM task 
+JOIN status ON task.status_id = status.id
+WHERE status.name != "Done";
+
+-- 5. Get all the tasks, sorted with the most recently created first
+SELECT *
```
Comment (sboutzen) [inline databases/week1/Home work lesson 1.sql:24]: Avoid * selection. In some cases `select *` can impact how rows are locked. It is better to explicitly write the column names you want

File: databases/week1/Home work lesson 1.sql [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+-- 6. Get the single most recently created task
+SELECT *
+ FROM task 
+ ORDER BY created DESC 
+ limit 1;
+ 
+ -- 7. Get the title and due date of all tasks where the title or description contains database
+ SELECT title, due_date
+ FROM task
+ WHERE title LIKE "%database%"
```
Comment (sboutzen) [inline databases/week1/Home work lesson 1.sql:37]: In mysql the `LIKE` is not case insensitive. Unless you are positive you only want this word with this exact case, you could use `WHERE LOWER(description) LIKE "%database%" OR LOWER(title) LIKE "%database%";`

File: databases/week1/Home work lesson 1.sql [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+ join task On task.status_id = status.id
+ group by status.name;
+ 
+ 
+ -- 10. Get the names of all statuses, sorted by the status with most tasks first
+ SELECT status.name, count(task.status_id) AS "Number of Tasks"
+ FROM status
+ join task ON task.status_id = status.id
+ group by status.name
+ order by count(task.status_id) desc;
```
Comment (sboutzen) [inline databases/week1/Home work lesson 1.sql]: I believe you could actually just order by `"Number of Tasks"`.

**Review by sboutzen — COMMENTED** [review]
Well done! Some small comments :)

File: databases/week1/Home work lesson 1.sql (hunk 4 above)
Comment (Hani-far) [reply databases/week1/Home work lesson 1.sql]: Thank you for the comments , It was very helpful

**Pull request merged**